types/iyzipay/iyzipay-tests.ts

```diff
@@ -0,0 +1,11 @@
+import Iyzipay from "iyzipay";
```

Comment:
your own readme uses CJS imports, please stick to CJS module usage

Reply:
Hello,

It is using Cjs because it was written in the past. Using I dont know Node 10 something.

I am updating test accordingly

Reply:
it will work fine with TS default tsconfig settings (esmoduleinterop). That's not required here to be most up-to-date, just most compatible. thanks for the change!